src/chords_generation/daria.py

```diff
@@ -105,14 +98,14 @@ def get_chords_daria(sounds: List[music.Sound],
                 max_chord = c
 
         t_c.append(music.Chord(max_chord.note,
-                               duration=half_bar_len/4,
+                               duration=constants.MIN_CHORD_DURATION,
```

Comment:
what is MIN_CHORD_DURATION?
This should be equal to width of window used for scoring.
This changes output of algorithm in, for example, lato_lato_C
left - before changes
right - after changes
![obraz](https://user-images.githubusercontent.com/38508437/100006229-7ab10f80-2dca-11eb-861e-521dbc1d799c.png)

Reply:
Yes, this is a parameter that changes the behaviour of the algorithm. It should be in the daria.py file indeed

Reply:
Done